packages/graphql-auth-transformer/src/resources.ts

```diff
@@ -1022,6 +1018,11 @@ identityClaim: "${rule.identityField || rule.identityClaim || DEFAULT_IDENTITY_F
         return block("Determine request authentication mode", expressions);
     }
 
+    public setStaticAuthVariable(field: FieldDefinitionNode): string {
```

Comment:
this should be called `getStaticAuthVariableName` as we are not setting the variable itself in this method

Reply:
changed to set